tests/e2e/specs/create_and_remove-existing-menu-test.js

```diff
@@ -0,0 +1,59 @@
+import { loginUser, visitAdminPage,activateTheme } from '@wordpress/e2e-test-utils';
+
+describe( 'Should create a new menu and remove existing menu', () => {
+
+    beforeEach( async () => {
+        await loginUser();
+	await activateTheme("twentytwentyone");
+    } );
+
+
+	it( 'Add a new menu', async () => {
+		
+        await visitAdminPage("nav-menus.php");
+        
+        //check if it's a first menu
+        await page.waitForSelector(".first-menu-message")
+
+        await page.type('#menu-name','New Menu');
+        
+        // select menu as a primary menu
+        await page.click("#locations-primary");
+
+        await page.click("#save_menu_footer");
+
+        await page.waitForSelector(".add-edit-menu-action");
+
+	} );
+
+
+    it( 'Remove existing menu', async () => {
+		
+	await visitAdminPage("nav-menus.php");
+
+        const createmenu = await page.$x("//a[normalize-space()='create a new menu']");
```

Comment:
Instead of using XPath here, maybe it will be easier to use a CSS selector? I'd do something like: 
`await page.click('.add-edit-menu-action a')`.